python/res/enkf/enkf_simulation_runner.py

```diff
@@ -116,5 +115,23 @@ def start_queue(self, run_context, job_queue):
             self.add_job(run_arg, self._enkf_main().resConfig(), job_queue, max_runtime)
 
         job_queue.submit_complete()
-        jqm = JobQueueManager(job_queue)
+        queue_validator = None
+        queue_validator_args = None
+        if self._enkf_main().analysisConfig().get_stop_long_running() and self._enkf_main().analysisConfig().minimum_required_realizations > 0:
+            queue_validator = EnkfSimulationRunner.stop_long_running_jobs
+            queue_validator_args = [self._enkf_main().analysisConfig().minimum_required_realizations]
```

Comment:
Could you use the [`functools.partial`](https://docs.python.org/2/library/functools.html#functools.partial) to bind the arguments to the function, so that we don't have to send both arguments and function to the queue manger?